### PR TITLE
tests: verify afterburn works with the old ignition.platform.id kernel argument ec2 and gce instead of aws and gcp

### PR DESCRIPTION
**Pull request by HuijingHei**
See https://issues.redhat.com/browse/OCPBUGS-7733

### CODE REVIEW

**Comment by HuijingHei** [comment]
Run passed on aws
```
$ kola run -d -p aws --aws-type t3.micro --aws-ami ami-05cbaa689092874a6 --aws-region us-east-1 ext.config.afterburn.platform-id

=== RUN   ext.config.afterburn.platform-id
2023-02-22T06:46:19Z platform/api/aws: instance ID not found, retrying: InvalidInstanceID.NotFound: The instance ID 'i-0bad1d2d3850390ff' does not exist
	status code: 400, request id: 6cd75ee5-dac1-4344-b2ce-865546244565
2023-02-22T06:46:19Z util: WaitUntilReady: checkFunction took 59.18087ms
2023-02-22T06:46:29Z util: WaitUntilReady: checkFunction took 155.675261ms
2023-02-22T06:46:29Z kola: Trying to StartMachine() i-0bad1d2d3850390ff
2023-02-22T06:46:52Z util: RetryUntilTimeout: f() took 23.099589709s
2023-02-22T06:47:03Z util: RetryUntilTimeout: f() took 55.125502ms
2023-02-22T06:47:33Z util: RetryUntilTimeout: f() took 20.034636484s
2023-02-22T06:47:43Z util: RetryUntilTimeout: f() took 773.695414ms
2023-02-22T06:47:44Z util: RetryUntilTimeout: f() took 345.321934ms
2023-02-22T06:47:48Z kola: Running kolet
2023-02-22T06:47:49Z kola: Starting kolet run-test-unit
2023-02-22T06:47:49Z kola: Reboot request with mark='rebooted'
2023-02-22T06:47:49Z kola: Waiting for reboot
2023-02-22T06:47:50Z util: RetryUntilTimeout: f() took 176.275452ms
2023-02-22T06:48:10Z util: RetryUntilTimeout: f() took 10.776413515s
2023-02-22T06:48:11Z util: RetryUntilTimeout: f() took 225.855072ms
2023-02-22T06:48:12Z kola: Reboot complete
2023-02-22T06:48:12Z kola: Starting kolet run-test-unit
2023-02-22T06:48:12Z kola: Setting AUTOPKGTEST_REBOOT_MARK='rebooted'
2023-02-22T06:48:18Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:48:18Z util: WaitUntilReady: checkFunction took 165.11645ms
2023-02-22T06:48:28Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:48:28Z util: WaitUntilReady: checkFunction took 154.966462ms
2023-02-22T06:48:38Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:48:38Z util: WaitUntilReady: checkFunction took 85.180209ms
2023-02-22T06:48:48Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:48:48Z util: WaitUntilReady: checkFunction took 129.997806ms
2023-02-22T06:48:58Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:48:58Z util: WaitUntilReady: checkFunction took 85.847329ms
2023-02-22T06:49:08Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:08Z util: WaitUntilReady: checkFunction took 95.547385ms
2023-02-22T06:49:18Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:18Z util: WaitUntilReady: checkFunction took 83.506018ms
2023-02-22T06:49:28Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:28Z util: WaitUntilReady: checkFunction took 87.766509ms
2023-02-22T06:49:38Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:38Z util: WaitUntilReady: checkFunction took 109.673632ms
2023-02-22T06:49:49Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:49Z util: WaitUntilReady: checkFunction took 85.06348ms
2023-02-22T06:49:59Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:49:59Z util: WaitUntilReady: checkFunction took 86.658692ms
2023-02-22T06:50:09Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:09Z util: WaitUntilReady: checkFunction took 85.914095ms
2023-02-22T06:50:19Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:19Z util: WaitUntilReady: checkFunction took 169.587196ms
2023-02-22T06:50:29Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:29Z util: WaitUntilReady: checkFunction took 87.999366ms
2023-02-22T06:50:39Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:39Z util: WaitUntilReady: checkFunction took 85.351279ms
2023-02-22T06:50:49Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:49Z util: WaitUntilReady: checkFunction took 86.774969ms
2023-02-22T06:50:59Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:50:59Z util: WaitUntilReady: checkFunction took 244.195745ms
2023-02-22T06:51:09Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:51:09Z util: WaitUntilReady: checkFunction took 90.703309ms
2023-02-22T06:51:20Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:51:20Z util: WaitUntilReady: checkFunction took 85.332534ms
2023-02-22T06:51:30Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:51:30Z util: WaitUntilReady: checkFunction took 103.62685ms
2023-02-22T06:51:40Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:51:40Z util: WaitUntilReady: checkFunction took 92.652084ms
2023-02-22T06:51:50Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:51:50Z util: WaitUntilReady: checkFunction took 95.200584ms
2023-02-22T06:52:00Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:52:00Z util: WaitUntilReady: checkFunction took 100.539539ms
2023-02-22T06:52:10Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:52:10Z util: WaitUntilReady: checkFunction took 127.196522ms
2023-02-22T06:52:20Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:52:20Z util: WaitUntilReady: checkFunction took 158.696719ms
2023-02-22T06:52:30Z platform/machine/aws: waiting for post-terminate console for i-0bad1d2d3850390ff
2023-02-22T06:52:30Z util: WaitUntilReady: checkFunction took 135.769423ms
2023-02-22T06:52:41Z util: WaitUntilReady: checkFunction took 190.7059ms
2023-02-22T06:52:41Z kola: Found systemd generator failure (/usr/lib/systemd/system-generators/podman-system-generator) on machine i-0bad1d2d3850390ff console
--- PASS: ext.config.afterburn.platform-id (383.21s)
PASS, output in tmp/kola/aws-2023-02-22-0646-299
```

File: tests/kola/afterburn/platform-id
Comment (travier) [inline tests/kola/afterburn/platform-id:4]: This should be below the special `## kola` section

File: tests/kola/afterburn/platform-id
Comment (travier) [inline tests/kola/afterburn/platform-id:5]:
```suggestion
##   platforms: aws gcp
#
# Make sure afterburn with old ignition.platform.id kernel argument ec2 and gce works.
# See https://issues.redhat.com/browse/OCPBUGS-7733
```

File: tests/kola/afterburn/platform-id
Comment (travier) [inline tests/kola/afterburn/platform-id:4]:
```suggestion
```

**Comment by bgilbert** [comment]
Thanks for doing this!

**Review by travier — APPROVED** [review]
/lgtm
CI here does not test this in AWS & GCP. Have you fully tested that in AWS & GCP manually?

If yes, then you can unhold this.
/hold

**Comment by HuijingHei** [comment]
> CI here does not test this in AWS & GCP. Have you fully tested that in AWS & GCP manually?

Thanks @travier for the heads up.
Just test on AWS, result is PASSED. Need to run on GCP manually.

File: tests/kola/afterburn/platform-id
Comment (dustymabe) [inline tests/kola/afterburn/platform-id:3]:
```suggestion
##   platforms: aws gce
```

We should probably eradicate all uses of gce from the mantle code base to reduce confusion here

Comment (HuijingHei) [reply tests/kola/afterburn/platform-id:3]: Nice catch! That sounds better to me.

**Comment by HuijingHei** [comment]
Run passed on GCP.
```
$ kola -E src/config -d --output-dir tmp/kola-gcp/ run -p=gce --gce-image=projects/rhcos-cloud/global/images/rhcos-xxx --gce-json-key=/srv/tool/gce.json --gce-project=openshift-rhcos-devel ext.config.afterburn.platform-id

=== RUN   ext.config.afterburn.platform-id
--- PASS: ext.config.afterburn.platform-id (112.07s)
PASS, output in tmp/kola-gcp
```

**Comment by HuijingHei** [comment]
Ready to review. Thanks!
/unhold

**Review by dustymabe — APPROVED** [review]
/approve

**Comment by dustymabe** [comment]
Maybe @bgilbert would want to make sure the test looks good before merge?

**Comment by HuijingHei** [comment]
Thanks all for the review!

**Pull request merged**
